### DISCUSSION TITLE
A-D snap tap dont work w/ FACEIT A.C?

### DISCUSSION THREAD

**Discussion by stvnkng** [wasiejen/Free-Snap-Tap#56] [General]
A-D snap tap dont work w/ FACEIT A.C?

**Comment by wasiejen** [comment]
Good morning @stvnkng :-),
I was made aware that Faceit updated there AC 4 or 5 days ago and now always has admin rights as far as I understood it. 
To my knowledge FST is not working well with any game or in this case AC that has admin rights. 

I don't know why exactly it is not working for Faceit but my guess would be that they are also listening for key and mouse events and get them directly - so there is no chance for FST to alter the input. And because they are listening they can ignore anything that is not coming directly from the input devices.

At the moment I do no know how to solve that.
